### DISCUSSION TITLE
provide an API ?

### DISCUSSION THREAD

**Discussion by Bitninejhkim** [apache/age#1787] [Ideas]
It would be great if you could provide an API. I would like to ask if you have any plans.

**Comment by markgomer** [comment]
That would be great! But I wonder how would it be different from the existing drivers 🤔

**Comment by dpdjvhxm** [comment]
@Bitninejhkim I think it is good to have something more, but provide API is too broad, I think it is good to think about more purpose of use, so might get some ideas from it
